xcm/xcm-builder/src/barriers.rs

```diff
@@ -23,6 +23,10 @@ use xcm_executor::traits::{OnResponse, ShouldExecute};
 use polkadot_parachain::primitives::IsSystem;
 
 /// Execution barrier that just takes `shallow_weight` from `weight_credit`.
+///
+/// Useful to allow XCM execution by local chain users via extrinsics.
+/// E.g. `pallet_xcm::reserve_asset_transfer` to transfer a reserve asset
```

Comment:
your line rewrapping is not consistent, I recommend using something like https://marketplace.visualstudio.com/items?itemName=stkb.rewrap#:~:text=The%20main%20Rewrap%20command%20is,column%20(default%20is%2080).

Reply:
or we wait for https://github.com/paritytech/substrate/pull/9490 :D

Reply:
Yeah, as long as there is no standard tooling I format how I think it looks good :-P